### ISSUE TITLE
The download link is invalid

### ISSUE THREAD

**Issue by Yuhuoo** [Championchess/Tri-Layer_Plugin_Occluded_Detection#1]
Thanks to your nice work, but I cannot download the .pkl using the download link. The download link is invalid. Can you help me?

**Comment by Yuhuoo** [comment]
> Thank you for your interest in our work. You can try to wget it using command line, and it should work.

Ok! Thanks for your reply, and I have another questions. What the LR can I set with 2 GPU and batch_size=4 during training ? How can I calcuate the LR ? Can I use the auto_scale_lr to set LR automatically? The auto_scale_lr is deleted from the default_runtime.py in the project

**Comment by Championchess** [comment]
As mentioned in the paper, we set the initial learning rate to be 0.000001 with batch_size=16. I have not tried to do with batch_size=4.
